bitnami/postgresql-repmgr/12/debian-11/rootfs/opt/bitnami/scripts/librepmgr.sh

```diff
@@ -560,13 +622,33 @@ EOF
 #########################
 repmgr_wait_primary_node() {
     local return_value=1
-    local -i timeout=60
-    local -i step=10
+    local -i timeout="${REPMGR_WAIT_PRIMARY_TIME}"
+    local -i step=$(( REPMGR_WAIT_PRIMARY_STEP + $(generate_random_string --type numeric --count 1) ))
     local -i max_tries=$((timeout / step))
     local schemata
     info "Waiting for primary node..."
     debug "Wait for schema $REPMGR_DATABASE.repmgr on '${REPMGR_CURRENT_PRIMARY_HOST}:${REPMGR_CURRENT_PRIMARY_PORT}', will try $max_tries times with $step delay seconds (TIMEOUT=$timeout)"
     for ((i = 0; i <= timeout; i += step)); do
+
+        if [[ -z "$REPMGR_CURRENT_PRIMARY_HOST" ]] ; then
+          # because primary could be down while standby turn on, so the primary host could not be get from upstream at that momement
+          # asking on each step help to give the primary the opportunity to come back
```

Comment:
Please fix typos in these comment lines.

Suggestion:
```
         # The primary node may not be available at the time the standby node turns on
         # Asking partner nodes who the primary node is to give it an opportunity to come back
```

Reply:
suggestion accepted